packages/server/src/fhir/accesspolicy.ts

```diff
@@ -32,9 +32,12 @@ export type PopulatedAccessPolicy = AccessPolicy & { resource: AccessPolicyResou
  * @returns A repository configured for the login details.
  */
 export async function getRepoForLogin(authState: AuthState, extendedMode?: boolean): Promise<Repository> {
```

Comment:
I'm trying to parse these changes....  Did anything change in this function, or is this all refactoring?

Reply:
Other than the change to read the correct project, the rest is minor refactoring